ledgereth/__main__.py

```diff
@@ -121,13 +136,22 @@ def send_value(dongle, args):
         dongle.close()
         sys.exit(1)
 
+    if not args.gasprice and not args.max_fee:
+        print(
+            "Either --gasprice or --max-fee must be provided", file=sys.stderr
+        )
+        dongle.close()
+        sys.exit(1)
```

Comment:
I believe there is an `ArgumentError` in `argparse` you could use.

Reply:
Do you think throwing exceptions is a better call than a clean exit?

Reply:
I think ideally the error should look the same as other argument-parsing related errors, like if you select a value not in an enum or something